planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/manager.cpp

```diff
@@ -440,6 +442,18 @@ void LaneChangeModuleManager::updateModuleParams(const std::vector<rclcpp::Param
       p->safety.collision_check.prediction_time_resolution);
     updateParam<double>(
       parameters, ns + "yaw_diff_threshold", p->safety.collision_check.th_yaw_diff);
+
+    auto th_incoming_object_yaw = p->safety.collision_check.th_incoming_object_yaw;
+    updateParam<double>(parameters, ns + "th_incoming_object_yaw", th_incoming_object_yaw);
+    if (th_incoming_object_yaw >= M_PI_2) {
+      p->safety.collision_check.th_incoming_object_yaw = th_incoming_object_yaw;
+    } else {
+      RCLCPP_WARN_THROTTLE(
+        node_->get_logger(), *node_->get_clock(), 5000,
+        "The value of th_incoming_object_yaw (%.3f rad) is less than the expected value (%.3f "
+        "rad).",
```

Comment:
(nitpick) it sounds like the parameter is expected to have a fixed value of `M_PI_2`.
```suggestion
        "The value of th_incoming_object_yaw (%.3f rad) is less than the minimum possible value (%.3f "
        "rad).",
```